hydra_agent/requests.py

```diff
@@ -0,0 +1,100 @@
+import urllib.request
+import json
+import logging
+from urllib.error import URLError, HTTPError
+from redis_proxy import RedisProxy
+from graphutils import GraphUtils
+from redisgraph import Graph, Node
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class Requests:
+
+    def __init__(self, entrypoint_url, redis_connection):
+        self.entrypoint_url = entrypoint_url
+        self.redis_connection = redis_connection
+        self.connection = redis_connection.get_connection()
+        self.vocabulary = 'vocab'
+        self.graph_utils = GraphUtils(redis_connection)
+        self.redis_graph = Graph("apidoc", redis_connection)
+
+    def get(self, url):
```

Comment:
As above, already implemented in `requests.Session.get`. This and all the other verbs just need to be overridden.